serving/docs/lmi/user_guides/trt_llm_user_guide.md

```diff
@@ -81,36 +81,43 @@ For `LMI` configurations, if we determine an issue with the configuration, we wi
+| option.smoothquant_per_token	                                 | >= 0.26.0   | Pass Through	      | This is only applied when `option.quantize` is set to `smoothquant`.  This enables choosing at run time a custom smoothquant scaling factor for each token. This is usally little slower and more accurate	                                                                                                                                                                                                                                                                                                                                                                                          | `true`, `false`. <br/> Default is `false`	                                                                                                                                  |
+| option.smoothquant_per_channel	                               | >= 0.26.0   | Pass Through	      | This is only applied when `option.quantize` is set to `smoothquant`.  This enables choosing at run time a custom smoothquant scaling factor for each channel. This is usally little slower and more accurate	                                                                                                                                                                                                                                                                                                                                                                                        | `true`, `false`. <br/> Default is `false`	                                                                                                                                  |
+| option.multi_query_mode	                                      | >= 0.26.0   | Pass Through	      | This is only needed when `option.quantize` is set to `smoothquant` . This is should be set for models that support multi-query-attention, for e.g llama-70b	                                                                                                                                                                                                                                                                                                                                                                                                                                         | `true`, `false`. <br/> Default is `false`	                                                                                                                                  |
+| Advanced parameters: AWQ	                                     |
```

Comment:
We also need to add Advanced parameters: FP8

Reply:
Added quantize, use_fp8_context_fmha, calib_size, and calib_batch_size; let me know if there is anything else I should pay attention to 👍